tools/src/dbgen/CLI.hs

```diff
@@ -34,7 +31,11 @@ data CLI = CLI
     -- ^ If true, print the stats for the `wallet-db`
     , queryMethod       :: Maybe Method
     -- ^ If true, generate a DB targeting mainnet.
```

Comment:
This comment seems to be incorrectly placed (it's probably me to blame here, but as we are reviewing the code let's change it 😉 )